packages/ui/src/components/keybind.css

```
@@ -6,12 +6,25 @@
   height: 20px;
   padding: 0 8px;
   border-radius: 2px;
-  background: var(--surface-base);
-  box-shadow: var(--shadow-xxs-border);
+  /* Spec: scrim/kbd colors inlined per DESIGN.md (no token). */
+  background: #f5f0e9;
+  border: 1px solid rgba(0, 0, 0, 0.08);
 
   font-family: var(--font-family-sans);
```

Stylelint (17.10.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)